string cmd
float64 start
float64 end
float64 time
float64 val
